libraries/EEPROM/EEPROM.cpp

```diff
@@ -75,7 +75,7 @@ void EEPROMClass::begin(size_t size) {
 
 void EEPROMClass::end() {
   if (!_size)
-    return;
+    return 0; //Added zero to increase readability for clients
```

Comment:
Nope, return type is void, so no return value.

Reply:
I am totally agree with u. I don't know what i thought last night when i typed this. Now i see for my code's condition is different.